_docs/_latest/concepts/best-practices.md

```diff
@@ -40,6 +40,135 @@ Scanning for external identities helps stop untrusted users from getting access
 When you add external users directly to a Cloud IAM policy, it's easier to audit their activities
 and do lifecycle management.
 
+**User Account Strategy**
+
+Ensure all employees are using a [Google Account from the enterprises Cloud
+Identity](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#use_corporate_login_credentials).
+This gives enterprise administrators full control over the lifecycle and
+security policies of that user account; deprovision when user leaves, enforce
+2SV, etc.
+
+**Project Naming Convention**
+
+Enterprises should agree on a standardised project naming convention. E.g.
+[system name]-[environment (dev, test, uat, stage, prod)] =>
+[costanalytics-dev](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#projects_are_identified_by_universally_unique_identifiers)
+Some documents recommend using [company tag]-[group tag] prefixed to the above,
+opinion today is that this means project names fall “out of date/stale” too
+fast as companies go through reorgs. Consider moving this information to
+project
+[labels](https://cloud.google.com/resource-manager/docs/creating-managing-labels#using_console).
+
+**Group Naming Convention**
+
+Groups are used to manage user membership in IAM policies. A consistent naming
+convention is recommended for enterprises. E.g. gcp-[system name]-[role type] =>
+gcp-costanalytics-prodsupport, gcp-costanalytics-analysts
+
+**Assign Roles to Groups**
+
+[Assign project roles to groups of
+users](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#assign_project_roles_to_groups_of_users).
+Rather than assign individual user accounts roles in IAM, setup a Group. User
+can then request to join a group (via their an automated and audited process)
+to gain access to the GCP projects.
+
+**Deeply Nested Sub Groups**
+
+Nested groups can [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)
```

Comment:
English a bit messy:
Group memberships can be hard to reason about when determining access across a deeply nested structure, and also propagation of projects to users will be delayed. 

 IAM Explain might help with this in the future, but for now as a best practice, one level
works best.